deploy/check_debezium.py

```
@@ -0,0 +1,21 @@
+Usage: python3 check_debezium.py <kafka_connect_url>
+Exit 0 if both are RUNNING, exit 1 otherwise.
+"""
+
+import json
+import sys
+import urllib.error
+import urllib.request
+
+url = f"{sys.argv[1]}/connectors/rbac-debezium/status"
+try:
+    with urllib.request.urlopen(url, timeout=5) as response:
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)